streamwebs_frontend/streamwebs_frontend/settings.py.dist

```diff
@@ -20,11 +20,11 @@ import sys
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 PROJECT_ROOT = os.path.join(BASE_DIR, 'streamwebs')
-STATIC_ROOT = os.path.join(PROJECT_ROOT, 'static')
+STATIC_ROOT = 'staticfiles'
```

Comment:
Using this setting in develop mode break the application? Have you tested that yourself?

Reply:
I'm not sure, I followed the usage guide on the github page.
https://github.com/kennethreitz/dj-static
The site works properly on my localhost.